src/store/Controller.cc

```diff
@@ -443,6 +443,12 @@ Store::Controller::transientsWriter(const StoreEntry &e) const
     return transients && e.hasTransients() && transients->isWriter(e);
 }
 
+bool
+Store::Controller::cacheWriter(const StoreEntry &e) const
+{
+    return transients ? transientsWriter(e) : true;
```

Comment:
If possible, please inline this method definition.

Reply:
Can we solve those linking problems by adding a stub for transientsWriter()? Or is there a bigger/deeper problem here?

Reply:
Done, I mislead you with my comment. This inlining worked without problems I was thinking about.